src/styles/card.scss

```diff
@@ -0,0 +1,143 @@
+$image_width: 198px;
+
+.product-card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: $image_width;
+  height: 401px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.product-card__image {
+  width: 160px;
+  height: 134px;
+  margin: 30px 0;
+  transition: 1s;
+}
+
+.product-card__image:hover {
+  padding: 9px 0;
+  margin: 15px 0 21px;
+  border: 3px solid #f3f3f3;
+  transform: scale(1.2);
+}
+
+.product-card__content-block {
+  padding: 0 15px;
+}
+
+.product-card__description-text {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 15px;
+  margin-top: 15px;
+  margin-bottom: 0;
+}
+
+.product-card__product-code {
+  width: 100%;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 11px;
+  line-height: 12px;
+}
+
+.review-block {
+  display: flex;
+  margin-top: 15px;
+}
+
+.review-block__stars {
+  display: flex;
+  width: 91px;
+  height: 15px;
+}
+
+.stars__star-item {
+  width: 13px;
+  height: 12px;
+  margin-right: 5px;
+  margin-left: 1px;
+  margin-top: 1px;
+  content: url("../images/star.svg");
+}
+
+.stars--5,
+.stars--1 :first-child,
+.stars--2 :nth-child(-n + 2),
+.stars--3 :nth-child(-n + 3),
+.stars--4 :nth-child(-n + 4) {
+  content: url("../images/star-active.svg");
+}
+
+.review-block__review-count {
+  width: 58px;
+}
+
+.product-card__product-price-block {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 18px;
+}
+
+.product-price-block__text,
+.review-block__review-count {
+  font-style: normal;
+  font-weight: normal;
+  text-align: right;
+  font-size: 11px;
+  line-height: 13px;
+  margin-left: 21px;
+}
+
+.product-price-block__text {
+  margin-left: 0;
+  margin-top: 6px;
+}
+
+.product-price-block__price {
+  font-style: normal;
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 19px;
+  text-align: right;
+  margin-top: 1px;

```

Comment:
😳 remove this

if something breaks after it, make sure you positioned everything in the right way, because this is not the solution